### PR TITLE
Close sidebar when going to concierge chat

### PR DESCRIPTION
**Pull request by tgolen**
### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/180850

### Tests / QA Steps
1. Have a bank account added to your account and a workspace created
2. Go to Settings > Workspace > Book travel
3. Click on the link to book travel with Concierge
4. Verify that you are taken to the concierge chat and the sidebar closes

### Tested On

- [x] Web
- [ ] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

### Screenshots
n/a

#### Web
n/a

#### Mobile Web
n/a

#### Desktop
n/a

#### iOS
n/a

#### Android
n/a

### CODE REVIEW

File: src/pages/workspace/travel/WorkspaceTravelPage.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const WorkspaceTravelPage = ({translate, route}) => (
     >
         {(hasVBA, policyID) => (
             <>
-                {!hasVBA ? (
+                {hasVBA ? (
```
Comment (Luke9389) [inline src/pages/workspace/travel/WorkspaceTravelPage.js:28]: Is this an unrelated fix?

**Comment by tgolen** [comment]
Updated

**Comment by Luke9389** [comment]
@parasharrajat I like that you're refactoring this behavior out into a CustomAction. Considering that this is a time-sensitive fix, could we merge this PR and then have you pull master on yours? That way we can have this quick fix done, and then give you the credit for abstracting this into more reusable behavior. Thoughts @tgolen?

**Review by Luke9389 — APPROVED** [review]
